operations/host_spawn.go

```diff
@@ -206,6 +207,10 @@ func hostModify() cli.Command {
 				Name:  extendFlagName,
 				Usage: "extend the expiration of a spawn host by `HOURS`",
 			},
+			cli.IntFlag{
+				Name:  temporaryExemptionFlagName,
+				Usage: "create or extend a temporary exemption from the host's sleep schedule by `HOURS`",
```

Comment:
Can we add a prompt or warning or something else that makes it clear to users that if they run it twice with "10", it will add 20 and not 10? Although it already mentions "or extend," I worry users might overlook this detail if they read quickly.

Reply:
I don't think there's an easy way to tell the user that they're extending without fetching the existing host data (which requires another step in the process), and I think it's a clear enough operation since it has the word "extend" in the flag name. The existing flag to extend an expirable host (right above this one) doesn't really warn either.